edx_proctoring/tests/test_api.py

```diff
@@ -1141,6 +1306,7 @@ def test_attempt_status_summary(self, status, expected):
         Assert that we get the expected status summaries
         """
 
+        set_runtime_service('grades', MockGradesService())
```

Comment:
No idea, I must have pasted this here by mistake. I'll remove it. Thanks for catching it.